test-configs/operator-wandb/default.yaml

```
@@ -9,6 +9,11 @@ global:
     GLOBAL_ADMIN_API_KEY: "local-123456789-123456789-123456789-1234"
     GORILLA_INSECURE_ALLOW_API_KEY_ADMIN_ACCESS: "true"
 
+prometheus:
+  install: true
+  instance: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)